src/test/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/integration/ClientsControllerIntTest.kt

```diff
@@ -89,6 +89,56 @@ class ClientsControllerIntTest : IntegrationTestBase() {
           )
         }
     }
+
```

Comment:
Do we also need some test cases for the sort order?

Reply:
Will add anther ticket to handle in UI
https://dsdmoj.atlassian.net/browse/HAAR-1684